grabbit/src/main/resources/META-INF/spring/client-batch-job.xml

```diff
@@ -32,16 +32,35 @@
     -->
 
     <import resource="client-batch-job-context.xml"/>
+
+    <import resource="client-namespace-step.xml" />
     <import resource="client-jcrNodes-step.xml" />
-    <import resource="client-preprocessor-step.xml" />
+    <import resource="client-deleteBeforeWrite-steps.xml" />
 
     <batch:job id="clientJob" xmlns="http://www.springframework.org/schema/batch" job-repository="clientJobRepository">
-        <batch:step id="clientPreprocessor" next="clientWorkflowOff">
-            <batch:tasklet ref="clientPreprocessTasklet" transaction-manager="clientTransactionManager"/>
-        </batch:step>
-        <batch:step id="clientWorkflowOff" next="clientJcrNodes">
-            <batch:tasklet ref="clientWorkflowOffTasklet" transaction-manager="clientTransactionManager"/>
+        <!--  Do any presync tasks concurrently that we can, e.g turning off workflows -->
+        <batch:split id="preSyncFlow" task-executor="asyncTaskExecutor" next="clientNamespaceSync">
+            <batch:flow>
+                <batch:step id="clientWorkflowOff">
```

Comment:
I do think that using the feature will make sense in some cases, but I still don't think that parallelizing `clientWorkflowOffTasklet` and `deleteBeforeWriteTasklet` makes sense (and may be its not safe either). 

For example : I think its perfectly reasonable to have a workflow that will trigger when a DAM Asset is removed (may be to remove `custom renditions` for that Asset or something).

For such a scenario, if we parallellize the two tasks above, it is possible that the nodes are removed before the corresponding workflows are turned off, which will result in undesirable effects.

I could however see `nameSpaceSync` and `clientWorkflowOff` tasks being parallelized. I think that will be safe, but the current structure I do not think is safe in all the cases.

What do others think @jdigger @jazeren1 @sikarwarvimal Paul?